edit/fleet.cattle.io.cluster.vue

```diff
@@ -35,8 +35,8 @@ export default {
       nc.metadata = {};
     }
 
-    nc.metadata.labels = nc._labels || {};
-    nc.metadata.annotations = nc._annotations || {};
+    nc.metadata.labels = nc.labels || {};
+    nc.metadata.annotations = nc.annotations || {};
```

Comment:
These aren't needed any longer - lines 38 and 39 can go

Reply:
fixed